pom.xml

```
@@ -4,101 +4,88 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.laba.solvd.jdbc</groupId>
-    <artifactId>Pharmacy</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <groupId>com.laba.solvd</groupId>
+    <artifactId>PharmacyJDBC_MyBatis</artifactId>
```

Don't use underscores here
```suggestion
    <artifactId>Pharmacy</artifactId>
```